### PR TITLE
Change logs for 1.0.3 and 1.0.4

### PR DESCRIPTION
**Pull request by robblovell**
## Description

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Technical improvement or removal of technical debt. 
- [x] Minor change: comments, documentation, trivial fixes

### CODE REVIEW

File: content/changelogs/cli.md [hunk 1]
```diff
@@ -2,6 +2,23 @@
...
+### Bug Fixes
+
+Environment monitor now checks process rather than the file to determine if a session is open (#2000)
+
+## Release Notes for v1.0.3
+
+### Bug Fixes
+
+Fixed so that the index.js and index.css files could be found on nested routes (#1978)
+Changed the newsletter contact form to use Google Forms (#1976)
```
Comment (conneryn) [inline content/changelogs/cli.md:20]: These are not related to CLI, so I think should be left out.  But there was one timeout fix that should probably be included.

```suggestion
* Increased timeouts for pod-less decoy (#1982)
```

File: content/changelogs/cli.md [hunk 2]
```diff
@@ -2,6 +2,23 @@
...
+
+### Bug Fixes
+
+Environment monitor now checks process rather than the file to determine if a session is open (#2000)
+
+## Release Notes for v1.0.3
+
+### Bug Fixes
+
+Fixed so that the index.js and index.css files could be found on nested routes (#1978)
```
Comment (conneryn) [inline content/changelogs/cli.md:19]:
```suggestion
```

File: content/changelogs/cli.md [hunk 3]
```diff
@@ -2,6 +2,23 @@
...
 
+## Release Notes for v1.0.4
+
+### New Features
+
+Support for intercepting all traffic (#1997)
+
+### Bug Fixes
+
+Environment monitor now checks process rather than the file to determine if a session is open (#2000)
```
Comment (conneryn) [inline content/changelogs/cli.md:13]: Bullet points.

```suggestion
* Environment monitor now checks process rather than the file to determine if a session is open (#2000)
```

File: content/changelogs/cli.md [hunk 4]
```diff
@@ -2,6 +2,23 @@
 
 This document contains the release notes for the CodeZero CLI.
 
+## Release Notes for v1.0.4
+
+### New Features
+
+Support for intercepting all traffic (#1997)
```
Comment (conneryn) [inline content/changelogs/cli.md:9]:
```suggestion
* Support for intercepting all traffic (#1997)
```

**Review by nsainaney — APPROVED** [review]
